plugins/smart_proxy_dns_route53/install

```diff
@@ -0,0 +1,2 @@
+debian/dns_route53.rb usr/share/foreman-proxy/bundler.d
+config/dns_route53.yml etc/foreman-proxy/settings.d
```

Comment:
Can we force permissions and ownership to something tighter here?  It's being installed as root:root 0644, but since it usually contains credentials, it should be root:foreman-proxy 0640, like the installer sets up.